core/scss/components/radio/radio.scss

```diff
@@ -37,6 +37,14 @@
             cursor: pointer;
           }
         }
+        + .#{$radio__class}--label {
+          .#{$radio__class}__label {
+            &:before {
+              color: $form-control__content;
+              cursor: pointer;
+            }
+          }
+        }
```

Comment:
Unclear why this PR has changes to the React radio component included, but these should probably be in a separate PR

Reply:
It was merge from master